### DISCUSSION TITLE
Alerting on services in Jaeger

### DISCUSSION THREAD

**Discussion by amrithaa1011** [jaegertracing/jaeger#3749] [Q&A]
Hello, 
Is there a way to implement alerting on services in Jaeger ?
For eg : I have a monitored service that takes 2 seconds each request. I want a slack alert to arrive if it takes more than 10 seconds, is there a plugin or could this be done from the jaeger itself ?
Thanks !

**Comment by yurishkuro** [comment]
You can do this with Prometheus, Jaeger does not implement alerting.
